lib/ae_mdw/txs.ex

```diff
@@ -37,27 +37,24 @@ defmodule AeMdw.Txs do
        ) do
     {block_hash, type, signed_tx, tx_rec} = Db.get_tx_data(tx_hash)
 
-    tx_map =
-      tx_rec
-      |> Format.to_raw_map(type)
-      |> put_in([:type], type)
+    header = :aec_db.get_header(block_hash)
+    %{"tx" => tx} = :aetx_sign.serialize_for_client(header, signed_tx)
 
     raw = %{
```

Comment:
Could we put this transaction code change in a separate PR to help keep tracking of the issues? (if it doesn't really relate).

Reply:
it does relate, because for every auction a transaction (last_bid) is rendered. This is why this change was introduced.

Reply:
Good, it's not used by the tx_controller.